compiler-rt/lib/mpk_untrusted/mpk_formatter.cpp

```diff
@@ -69,30 +80,31 @@ bool writeUniqueFile(std::set<AllocSite> &faultSet) {
     }
   }
 
-  std::ofstream OS;
-  if (!makeUniqueFile(TestDirectory + "/faulting-allocs-%%%%%%%.json", OS))
+  auto OS = makeUniqueStream(TestDirectory, "faulting-allocs", "json");
```

Comment:
its probably better to  get the option, check it, and then unwrap the stream and use it directly instead of calling `getValue()` repeatedly.